tests/tests_create_raid_pool_then_remove.yml

```diff
@@ -15,11 +15,12 @@
     - include_role:
         name: storage
 
+    # For testing raid 0
     - include_tasks: get_unused_disk.yml
       vars:
         max_return: 2
 
-    - name: Create a RAID0 device
+    - name: Create a RAID0 device mounted on "{{ mount_location1 }}"
```

Comment:
There are three different mount locations here.

Reply:
the simplest would be to omit the change then.